Core/Filesystem/src/core/nand_core.c

```diff
@@ -33,7 +33,7 @@ static void _increment_seek(PhysicalAddrs *addr, int size);
 static void _increment_block(PhysicalAddrs *addr);
 static void find_good_block(PhysicalAddrs *addr);
 
-#define RESERVED_BLOCK_CNT 0
+#define RESERVED_BLOCK_CNT 1
```

Comment:
I thought this was only used on the wrap-around, to skip the first N blocks. What can that break?